src/components/Icon/Icon.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../assets/variables.scss";
+
+@mixin icons{
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+.icon{
```

Comment:
pls check if file requires formatting 😉 
```suggestion
.icon {
```

Reply:
That's prove that it is something wrong with pre-push `Stylelint` function. I will fix formatting in this file and in future I will add github actions with appropriate settings for pre-push `ESLint` and `Stylelint`.